src/main/kotlin/org/wycliffeassociates/otter/common/domain/md/ParseMd.kt

```diff
@@ -0,0 +1,54 @@
+package org.wycliffeassociates.otter.common.domain.md
+
+import java.io.BufferedReader
+
+// TODO: Add Help type enum to HelpResource? (tn, tq)
+data class HelpResource(var title: String, var body: String)
+
+class ParseMd {
+
+    companion object {
+
+        private val isTitleRegex = Regex("^#+\\s*[^#\\s]+")
+        private val titleTextRegex = Regex("^#+\\s*")
+
+        fun parse(reader: BufferedReader): List<HelpResource> {
```

Comment:
I'm just being picky too, and curious, but wouldn't it be better if the parser takes in the `File` that it needs to parse as opposed to a `BufferedReader`? In practice I suppose I wouldn't want to have to set up the reader every time i wanted to parse some MD. I'd think the reader would be contained in the parser and i wouldn't have to worry about that

Reply:
A BufferedReader can be created from an InputStream which is the preferred way of accessing resources files, streaming a file from a url, etc. File would add restrictions to how the parser could be used, so I'm good with keeping it a BufferedReader.

Good question!

Reply:
@natibekele @sarabiaj This was actually a result of a conversation with Craig about unit testing. If the parser takes in a file, the unit test also has to supply a file, which is problematic since 1. it takes longer during unit test to read from a file, and 2. we don't want our unit tests to be dependent on files in specific directories. Thought I'd share this, since I thought Craig made some really good points!